client/fixtures/productDetail.ts

```diff
@@ -890,8 +890,77 @@ export const membership: ProductDetail = {
 		currentPlans: [],
 		futurePlans: [],
 		readerType: 'Direct',
-		accountId: '8ad088718219a6b601821bbe9e6210f2',
+		accountId: '1',
+	},
+};
+
+export const membership: ProductDetail = {
```

Comment:
With the multiple variations of membership could we rename this to supporter membership or something similar?

Reply:
could do, named it `membership` just to keep the references in the other tests the same